JapaneseFilter/sections/specific.txt

```diff
@@ -39,6 +39,7 @@ yutura.net,goetheweb.jp,mono-money.com,tetsudo-ch.com,home.kingsoft.jp,coco-tte.
 !
 !
 !
+mbs.jp###pagewrap > .common-wrap--article ~ .common-wrap:not(.footer-banner)
```

Comment:
I guess this is for Annoyances filter - self-promo. @Yuki2718 what do you think?

Reply:
It's popIn leftover, should be Annoyances (popIn section) and on my end `mbs.jp##.common-wrap--ranking` is enough.

Reply:
Done.